docs/development.md

```diff
@@ -44,9 +51,9 @@
     $ pip3 install cython
     ```
 
-7. Install [tesseract](https://github.com/tesseract-ocr/tesseract/wiki) OCR framework.
+8. Install [tesseract](https://github.com/tesseract-ocr/tesseract/wiki) OCR framework.
```

Comment:
optional or mandatory?

Reply:
Depends, if wants to work on OCR... then mandatory

Reply:
So it's optional :)